workflow/snakemake_rules/core.smk

```diff
@@ -22,9 +22,8 @@ rule wrangle_metadata:
         strain_id=lambda w: config.get("strain_id_field", "strain"),
     shell:
         """
-        python3 scripts/wrangle_metadata.py --metadata {input.metadata} \
-                    --strain-id {params.strain_id} \
-                    --output {output.metadata}
+        csvtk -t rename -f strain -n strain_original {input.metadata} \
+            | csvtk -t mutate -f {params.strain_id} -n strain > {output.metadata}
```

Comment:
_(non-blocking)_

```suggestion
            | csvtk mutate -t -f {params.strain_id} -n strain > {output.metadata}
```

At first, `csvtk -t mutate` read to me as if `-t` was given the `mutate` value and I wondered what that meant. Only after looking at the [docs](https://bioinf.shenwei.me/csvtk/usage/#mutate) did I understand that `mutate` is a subcommand. Those docs are inconsistent though – both `csvtk -t -H filter` and `csvtk filter2 -H -t` are used for examples.